rtcpeerconnection.go

```diff
@@ -127,9 +128,28 @@ func (r *RTCPeerConnection) CreateAnswer() error {
 			if server.serverType() != RTCServerTypeSTUN {
 				continue
 			}
-			// TODO connect to STUN server
-			_ = id
-			_ = server
+
+			for _, url := range server.URLs {
+				// TODO if one of the URLs does not work we should just ignore it.
+				client, err := stun.NewClient("udp", url, time.Second*5)
```

Comment:
This is so close to being able to do STUN, nice work!

You should be able to just do `network.NewPort("0.0.0.0:" + addr.Port, ....`

All you need to is update the typ of the candidate, and that should be it! You can get the examples from [here](https://webrtchacks.com/sdp-anatomy/)